Geodata_CityGML/Geodata_CityGML_shacl.ttl

```diff
@@ -0,0 +1,57 @@
+@prefix Geodata_CityGML: <https://github.com/GAIA-X4PLC-AAD/map-and-scenario-data/tools/ontologie_creator/ontologies/Geodata_CityGML_ontology/> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+Geodata_CityGML:Asset_Shape a sh:NodeShape ;
+    sh:property [ skos:example "Building, Bridge, Vegetation, .. " ;
+            sh:datatype xsd:string ;
+            sh:description "Included CityGML top-level feature types"@en ;
+            sh:message "Validation of featureTypes failed!"@en ;
+            sh:minCount "0"^^xsd:unsignedInt ;
+            sh:name "featureTypes"@en ;
+            sh:path Geodata_CityGML:Geodata_CityGML_quality_featureTypes ],
+        [ skos:example "2011-11-11 00:00:00" ;
+            sh:datatype xsd:string ;
+            sh:description "Creation Date"@en ;
+            sh:message "Validation of creationDate failed!"@en ;
+            sh:minCount "0"^^xsd:unsignedInt ;
+            sh:name "creationDate"@en ;
+            sh:path Geodata_CityGML:Geodata_CityGML_quality_creationDate ],
```

Comment:
attributes in camelCase.
Please refactor like in environment-model to have nested shapes rather than a plain list of attributes.

Please change it for the others, too.